### PR TITLE
Stripe: Move Multibanco to payment intents

### PR DESCRIPTION
**Pull request by raphaelm**

### CODE REVIEW

File: src/pretix/plugins/stripe/templates/pretixplugins/stripe/pending.html [hunk 1]
```diff
@@ -1,10 +1,43 @@
...
-    <p>{% blocktrans trimmed %}
-        We're waiting for an answer from the payment provider regarding your payment. Please contact us if this
-        takes more than a few days.
-    {% endblocktrans %}</p>
+    {% if payment_info.next_action.type == "multibanco_display_details" %}
+        <p><strong>{% trans "Payment instructions" %}:</strong></p>
+        <ol>
+            <li>
+                {% blocktrans trimmed %}
+                    In your online bank account or from an ATM, choose “Payment and other services”.
```
Comment (pc-coholic) [inline src/pretix/plugins/stripe/templates/pretixplugins/stripe/pending.html:10]: Those are some non-standard quotes...

File: src/pretix/plugins/stripe/templates/pretixplugins/stripe/pending.html [hunk 2]
```diff
@@ -1,10 +1,43 @@
...
+                {% endblocktrans %}
+            </li>
+            <li>
+                {% blocktrans trimmed %}
+                    Click “Payments of services/shopping”.
```
Comment (pc-coholic) [inline src/pretix/plugins/stripe/templates/pretixplugins/stripe/pending.html:15]: As are these.

**Review by pc-coholic — APPROVED** [review]
LGTM but not tested due to local branch-salad...

File: src/pretix/plugins/stripe/templates/pretixplugins/stripe/pending.html (hunk 1 above)
Comment (raphaelm) [inline src/pretix/plugins/stripe/templates/pretixplugins/stripe/pending.html:10]:
```suggestion
                    In your online bank account or from an ATM, choose "Payment and other services".
```

File: src/pretix/plugins/stripe/templates/pretixplugins/stripe/pending.html (hunk 2 above)
Comment (raphaelm) [inline src/pretix/plugins/stripe/templates/pretixplugins/stripe/pending.html:15]:
```suggestion
                    Click "Payments of services/shopping".
```

**Pull request merged**